index.html

```diff
@@ -1106,8 +1308,13 @@ <h3>Role Mapping Table</h3>
 								</td>
 								<td><p>Expose <code>ROLE_HEADING</code> and object attribute
 											<code>xml-roles:doc-subtitle</code>. </p></td>
-								<td>AXRole: <code>AXHeading</code><br /> AXSubrole: <code>&lt;nil&gt;</code><br />
-									AXRoleDescription: <code>'heading'</code>
+								<td>
+									<ul>
+										<li>AXRole: <code>AXHeading</code></li>
+										<li>AXSubrole: <code>AXSubtitle</code></li>
```

Comment:
TBD… Need subrole confirmation from platform API owners.

Reply:
Confirming as-is with new subrole.

Reply:
+1